Python/iSenseAPI.py

```diff
@@ -15,13 +17,19 @@
 # Gets info about a project, if recur is true it returns extra info
 def getProject(projID, recur=False):
     url = BASE_URL + 'projects/' + str(projID) +'?recur=' + str(recur).lower()
-    return requests.get(url)
+    request = requests.get(url)
+
+    dictionary = json.loads(request.text)
+    return dictionary
```

Comment:
What happens if the project isn't found?  Ideally, this should handle 404s and other easily predictably HTTP responses (like POST methods that require authorization should be able to handle responses telling the user that they're unauthorized).